creator-node/package.json

```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "start": "nodemon src/index.js | ./node_modules/.bin/bunyan || exit 0",
     "test": "./scripts/run-tests.sh",
+    "coverage": "nyc npm run test",
+    "report": "nyc report --reporter=html",
```

Comment:
would be good to add a comment *somewhere* about what purpose these commands serve